db_management/db_online.rst

```diff
@@ -55,3 +85,36 @@ database.
 
     .. image:: media/dup_expires.png
         :align: center
+
+.. _delete_online_database:
+
+Deleting a Database
+===================
+
+You can only delete databases of which you are the Administrator.
```

Comment:
`administrator` lowercase, he has already too much power, don't make him feel more important than he is ;-)

Reply:
You mean `ADMINISTRATOR`?